src/sass/accordion/_base.scss

```diff
@@ -38,18 +38,32 @@
 
   &__toggle-text {
     color: $color-black-500;
-    font-size: $ts-18;
-    font-weight: $font-weight-bold;
+    font-size: $ts-20;
+    font-weight: $font-weight-regular;
     line-height: 1.3;
     padding-right: $spacing-sm;
   }
 
   &__toggle-icon {
     color: $color-black-400;
     margin-left: auto;
+    position: relative;
+    top: 4px;
   }
 
   &__panel {
-    padding: $spacing-md;
+    padding: $spacing-xs $spacing-sm $spacing-md $spacing-sm;
+  }
+
+  &__panel > * {
+    margin-top: 0;
+  }
+
+  &__panel > * + * {
+    margin-top: 1.5rem;
+  }
+
+  &__panel > *:last-child {
```

Comment:
After removing the bottom margin for all elements in the block above, this can be deleted.